From foo@baz Tue Aug 18 09:31:28 PDT 2009
Date: Tue, 18 Aug 2009 09:31:28 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: USB: storage: increase the bcd range in Sony's bad device table.
Patch-mainline: 2.6.32
References: bnc#466554

Another Sony device in the wild is needing this hack.  You would think
they would fix their firmware one of these days...

Cc: Maciej Pilichowski <bluedzins@wp.pl>
Cc: stable <stable@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/usb/storage/unusual_devs.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/storage/unusual_devs.h
+++ b/drivers/usb/storage/unusual_devs.h
@@ -568,7 +568,7 @@ UNUSUAL_DEV(  0x052b, 0x1911, 0x0100, 0x
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
 		US_FL_IGNORE_RESIDUE ),
 
-UNUSUAL_DEV(  0x054c, 0x0010, 0x0106, 0x0450,
+UNUSUAL_DEV(  0x054c, 0x0010, 0x0106, 0x0500,
 		"Sony",
 		"DSC-S30/S70/S75/505V/F505/F707/F717/P8",
 		US_SC_SCSI, US_PR_DEVICE, NULL,
